docs/docs/rest-api/public/api/v2/types/mesosContainer.raml

```diff
@@ -1,64 +1,82 @@
-#%RAML 1.0 DataType
-type: object
-(scalaType): MesosContainer
-description: Mesos Container
-properties:
-  name:
-    type: !include name.raml
-    description: The name of this container
-  exec?:
+#%RAML 1.0 Library
+uses:
+  artifacts: artifact.raml
+  env: environmentVariable.raml
+  health: healthCheck.raml
+  labels: label.raml
+  mesosCommand: mesosCommand.raml
+  network: network.raml
+  resources: resources.raml
+  secrets: secrets.raml
```

Comment:
actually let's just declare these at the pod level. v2-style envvvars in the container can reference them from there